src/components/button-toggle/button-toggle.style.ts

```diff
@@ -65,6 +67,40 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
   box-sizing: border-box;
   max-width: 100%;
 
+  ${({ variant }) =>
+    variant === "minor" &&
+    css`
+      border: 1px solid var(--colorsActionMinor500);
+
+      &&&&&& {
```

Comment:
Comment: This doesn't feel like the right approach to me, it feels quite brittle and we could end up needing to raise specificity to ensure this works. I think it would be better to move the border radius styling into the `StyleButtonToggleWrapper` element and pass `variant` into that so we can set the radius accordingly.

Reply:
```
const StyledButtonToggleWrapper = styled.div<StyledButtonToggleWrapperProps>`
  display: inline-block;
  vertical-align: middle;

  ${({ grouped, variant }) =>
    css`
      ${!grouped &&
      css`
        &&&& {
          ${StyledButtonToggle} {
            border-radius: ${variant === "default"
              ? "var(--borderRadius400)"
              : "var(--borderRadius050)"};
          }
        }
      `}

      ${grouped &&
      css`
        &&&& {
          :first-of-type {
            ${StyledButtonToggle} {
              border-top-left-radius: ${variant === "default"
                ? "var(--borderRadius400)"
                : "var(--borderRadius050)"};
              border-bottom-left-radius: ${variant === "default"
                ? "var(--borderRadius400)"
                : "var(--borderRadius050)"};
            }
          }

          :last-of-type {
            ${StyledButtonToggle} {
              border-top-right-radius: ${variant === "default"
                ? "var(--borderRadius400)"
                : "var(--borderRadius050)"};
              border-bottom-right-radius: ${variant === "default"
                ? "var(--borderRadius400)"
                : "var(--borderRadius050)"};
            }
          }
        }
      `}
    `}

  &:not(:first-of-type) {
    margin-left: 8px;
  }

  ${({ grouped }) =>
    grouped &&
    css`
      &:not(:first-of-type) {
        margin-left: -1px;
      }
    `};
`;
```